lib/flip_card.dart

```diff
@@ -101,6 +103,7 @@ class FlipCard extends StatefulWidget {
     this.alignment = Alignment.center,
     this.fill = Fill.none,
     this.side = CardSide.FRONT,
+    this.builder,
   }) : super(key: key);
```

Comment:
Add asserts to make sure that either `front` and `back` are defined OR `builder`, but not both

Reply:
in current api front and back are required, builder is just to wrap front and back. Asserts unnecessary here.